docs: Update repository documents (#554)
pulsate-dev/pulsate: CONTRIBUTING.md

  - [Applying migration](#applying-migration)
  - [Rolling back a migration](#rolling-back-a-migration)

**Before "Contribution"**:

- All Contributors and Maintainers are required to follow the [Code of Conduct](https://github.com/pulsate-dev/.github?tab=coc-ov-file).

> - The Pulsate model and design are summarised in [Pulsate Specification](https://spec.pulsate.dev). Please read it when developing.

Reviewer: ```suggestion
- See [Pulsate Specification](https://spec.pulsate.dev)  about the Pulsate's model and design.
```
そのドキュメントは要約しているというより詳細に述べているので, 参照してもらいたいことも考えるとこのような表現でどうでしょうか.
